notebooks/00.00-Preface.ipynb

```diff
@@ -22,11 +22,11 @@
    "cell_type": "markdown",
```

Comment:
<p>test comment to check notification delay</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit2rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>